connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/actions/OpenApiClientRequestActionBuilder.java

```diff
@@ -56,6 +56,16 @@ public OpenApiClientRequestActionBuilder(OpenApiSpecification openApiSpec, Strin
     public OpenApiClientRequestActionBuilder(HttpMessage httpMessage, OpenApiSpecification openApiSpec,
                                              String operationId) {
         super(new OpenApiClientRequestMessageBuilder(httpMessage, openApiSpec, operationId), httpMessage);
+
+        openApiRequestValidationProcessor = new OpenApiRequestValidationProcessor(openApiSpec, operationId);
+        process(openApiRequestValidationProcessor);
+    }
+
+    public OpenApiClientRequestActionBuilder disableOasValidation(boolean b) {
```

Comment:
since this method is called "disable" I would just assume it disables validation (no matter what). I would not expect a parameter for sure. thoughts?